app/views/notifier/changeset_comment_notification.html.erb

```diff
@@ -1,20 +1,73 @@
+                  <h1 style="font-size: 18px; font-weight: 600; margin: 0; text-align: left">OpenStreetMap</h1>
+                </a>
+              </td>
+            </tr>
+            <tr>
+              <td colspan="2">
+                <table style="background-color: #fff; color: #222; border: solid 1px #ccc; border-collapse: separate">
+                  <tr>
+                    <td style="text-align: left; padding: 15px 15px 5px 15px">
+                      <p style="margin: 0">
+                        <% if @owner %>
+                          <%= t "notifier.changeset_comment_notification.commented.your_changeset_html", :commenter => @commenter, :commenter_url => @commenter_url, :time => @time %>
+                        <% else %>
+                          <%= t "notifier.changeset_comment_notification.commented.commented_changeset_html", :commenter => @commenter, :commenter_url => @commenter_url, :time => @time, :changeset_author => @changeset_author %>
+                        <% end %>
+                        <% if @changeset_comment %>
+                          <%= t "notifier.changeset_comment_notification.commented.partial_changeset_with_comment_html", :changeset_comment => @changeset_comment %>
+                        <% else %>
+                          <%= t "notifier.changeset_comment_notification.commented.partial_changeset_without_comment" %>
+                        <% end %>
+                      </p>
+                      <table style="font-size: 15px; font-style: italic; margin: 15px; background-color: #eee; width: 520px">
+                        <tr>
+                          <td style="width: 50px; vertical-align: top; padding: 15px">
+                            <a href="<%= @commenter_url %>" target="_blank"><%= image_tag attachments["avatar.png"].url, alt: @commenter %></a>
+                          </td>
+                          <td style="text-align: left; vertical-align: top; padding-right: 10px">
+                            <%= @comment.to_html %>
+                          </td>
+                        </tr>
+                      </table>
+                      <p>
+                        <%= raw t 'notifier.changeset_comment_notification.details_html', :url => link_to(@changeset_url, @changeset_url) %>
+                      </p>
+                    </td>
+                  </tr>
+                </table>
+              </td>
+            </tr>
+          </table>
+        </td>
+      </tr>
+      <tr>
+        <td style="text-align: center; font-size: 11px">
+          <p>
+            <%= t 'notifier.changeset_comment_notification.unsubscribe_html', :url => @changeset_url %>
+          </p>
+          <p style="margin-bottom: 10px">
+            <a href="<%= @root_url %>" target="_blank" style="color: #222">OpenStreetMap</a>
+          </p>
+        </td>
+      </tr>
+    </table>
+  </body>
+</html>
```

Comment:
There's a lot of boiler plate here that makes this very hard to read, and if you're planning to expand this to other emails then I then think you need to look for a way to separate it out so that it's not duplicated all over the shop.

Is there something clever we can do here with a layout?